src/Input/InputEvents.cpp

```diff
@@ -88,7 +87,7 @@ namespace InputEvents
 
   static unsigned MenuTimeOut = 0;
   
-  static unsigned LockScreen = 0; // LOCKSCREEN
+  bool LockScreen = 0;
```

Comment:
There's still no documentation on what this variable means.

Reply:
It's to store the enabling status of lockscreen function, now I commented it in the code